### ISSUE TITLE
Issues while creating a new bot

### ISSUE THREAD

**Issue by upgundecha** [claudiajs/claudia-bot-builder#22]
I am following tutorial at https://claudiajs.com/tutorials/hello-world-chatbot.html

After executing 

```
claudia create --region us-east-1 --api-module bot
```

I see following message

```
project name is missing. please specify with --name
```

When I specify the name 
```
claudia create --region us-east-1 --api-module bot --name mybot
```

I see following message. The newly created bot is not deployed.
```
package.json does not contain the files property
```

**Comment by stojanovic** [comment]
@upgundecha you did npm init first? Can you share your package.json file?

**Comment by upgundecha** [comment]
@stojanovic Just now this issue is resolved after upgrading Node v5.10.1 to Node v6.3.1

I did follow exact steps starting with npm init. Here is my package.json for reference

```
{
  "name": "mybot",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "claudia": "^1.6.0",
    "claudia-bot-builder": "^1.4.0",
    "huh": "^1.1.4"
  }
}
```

**Comment by stojanovic** [comment]
Hm, it can be some strange npm/node issue.
Anyway, for me it works fine on 4.3.2 (same version AWS runs on Lambda).

Btw, no need to set claudia as dependency, save it as dev dependency (`npm i claudia -D`). Dependencies are sent to Lambda, dev dependencies are not, and you don't need to send claudia, just bot builder.
